recipes/sdl/all/conanfile.py

```diff
@@ -233,6 +238,8 @@ def _configure_cmake(self):
             elif self.settings.os == "Windows":
                 self._cmake.definitions["DIRECTX"] = self.options.directx
 
+            self._cmake.definitions["HAVE_LIBUNWIND_H"] = self.options.get_safe("libunwind")
+
```

Comment:
could you move this definition in the `if self.settings.os == "Linux"` branch please?